### ISSUE TITLE
Image still not shown in mobile even with relative path

### ISSUE THREAD

**Issue by aifusenno1** [typora/typora-issues#3767]
My .md files are in iCloud. I have used relative path for images and they are inside iCloud as well, but I'm still not able to render any of the images on iOS. I have tried *Pretext* and *MDNotes*.

#### Code
```
## TEST
![image](image1.png)
<img src="Test.assets/image2.png" alt="image2" style="zoom:33%;" />
```
#### Filesystem
<img width="345" alt="image" src="https://user-images.githubusercontent.com/9910987/89503754-d738e200-d77b-11ea-936a-80025215700d.png">

#### In MDNotes app
<img width="345" alt="IMG_0329" src="https://user-images.githubusercontent.com/9910987/89504117-62b27300-d77c-11ea-9308-14089ac3caba.jpeg" style="zoom:30%;">
The two "images" you see are just the alt or the image name.

This seems like a super common problem that other people could have encountered, yet I failed to find a relevant issue.
In this [comment](https://github.com/typora/typora-issues/issues/2642#issuecomment-513536639) @abnerlee has mentioned about data sync. I did exactly the same but still cannot see the images.

Is this something related to iCloud?

**Comment by abnerlee** [comment]
What about "![image](./image1.png)"

And could you confirm if Pretext and MDNotes supports relative image or not, or how to embed local images in their renders?

**Comment by aifusenno1** [comment]
Not quite sure what you meant, as this is what I see. <img width="182" alt="image" src="https://user-images.githubusercontent.com/9910987/89593956-735afb80-d805-11ea-8fb4-4c99257b3b58.png">

But for the test example I showed, both images were not found. Those "image-like" icons you see are just the name of the image, not the actual image.

I have also tried MWeb, same thing happened. 
Not able to find detailed docs for the app.

**But my guess is, whenever opening a file from an external folder (meaning not a built-in storage space for the app), such as a md file in iCloud, the iOS apps are only able to see that file itself, not the entire directory.**

<img width="497" alt="image" src="https://user-images.githubusercontent.com/9910987/89594610-29731500-d807-11ea-89c3-79d2efe7a7de.png">

You see how the images are grayed out, and you are only able to open a file, not the entire folder.

I was assuming storing everything in the cloud would work seamlessly because on [your website](http://support.typora.io/Sync/)  it's not mentioned. 
Surprised that nobody has brought this up.
How does everyone else store the images while supporting viewing notes on mobile devices?
